src/com/TTT/Board.java

```diff
@@ -91,4 +90,26 @@ private int maximumColumnValue(int columnNumber) {
         return ((sideLength() * sideLength()) - sideLength()) + columnNumber;
     }
 
+    public List<Integer> availableMoves() {
+        List<Integer> availableMoves = new ArrayList<>();
+
+        for (int i = 0; i < size(); i++) {
+            if (!isOccupied(i)) {
```

Comment:
To improve readability, it might be nicer to remove the negative, and have a method `isVacant` instead.